{
  "Header": {
    "Original": "CVE-2022-45199 (Pillow before 9.3.0 allows denial of service via SAMPLESPERPIXEL. ...)",
    "Line": 8356,
    "ID": "CVE-2022-45199",
    "Description": "(Pillow before 9.3.0 allows denial of service via SAMPLESPERPIXEL. ...)"
  },
  "Annotations": [
    {
      "Original": "- pillow \u003cunfixed\u003e (bug #1024512)",
      "Line": 8357,
      "Type": "package",
      "Package": "pillow",
      "Kind": "unfixed",
      "BugNo": 1024512
    },
    {
      "Original": "[bullseye] - pillow \u003cnot-affected\u003e (Vulnerable code not present, introduced in 9.2.0)",
      "Line": 8358,
      "Type": "package",
      "Release": "bullseye",
      "Package": "pillow",
      "Kind": "not-affected",
      "Description": "Vulnerable code not present, introduced in 9.2.0"
    },
    {
      "Original": "[buster] - pillow \u003cnot-affected\u003e (Vulnerable code not present, introduced in 9.2.0)",
      "Line": 8359,
      "Type": "package",
      "Release": "buster",
      "Package": "pillow",
      "Kind": "not-affected",
      "Description": "Vulnerable code not present, introduced in 9.2.0"
    },
    {
      "Original": "NOTE: https://github.com/python-pillow/Pillow/commit/2444cddab2f83f28687c7c20871574acbb6dbcf3 (9.3.0)",
      "Line": 8360,
      "Type": "NOTE",
      "Description": "https://github.com/python-pillow/Pillow/commit/2444cddab2f83f28687c7c20871574acbb6dbcf3 (9.3.0)"
    },
    {
      "Original": "NOTE: https://github.com/python-pillow/Pillow/pull/6700",
      "Line": 8361,
      "Type": "NOTE",
      "Description": "https://github.com/python-pillow/Pillow/pull/6700"
    }
  ]
}